src/marionette.sliding-view.js

```diff
@@ -24,6 +24,9 @@ Mn.SlidingView = Mn.CollectionView.extend({
 
     // Get our initial boundaries, and then update the collection
     this._lowerBound = _.result(this, 'initialLowerBound');
+    if (typeof this.initialUpperBound  === 'function') {
+      this.initialUpperBound = this.initialUpperBound.bind(this, this._lowerBound);
+    }
```

Comment:
it looks like they don't plan to add arguments to underscore's `_.result` any time soon. Should I just add in the check's inline or make some private helper method on the view?